packages/zoe/src/zoeService/types.js

```diff
@@ -186,13 +186,17 @@
  */
 
 /**
- * @typedef {Object} InvitationDetails
+ * @typedef {Object} StandardInvitationDetails
```

Comment:
Is this so something else can have variant invitationDetails? I'd prefer the common case have the simpler name.

Reply:
Nope, this is just that InvitationDetails can take custom details. The type was wrong before. StandardInvitationDetails will never be used alone

Reply:
Just now, reading the types, this helped me understand.  I would have come up with `Base*` rather than `Standard*`, but `Standard*` vs `Custom*` works better.